pkg/iac-providers/terraform/commons/extract-container-images.go

```diff
@@ -126,7 +126,7 @@ func getContainersFromhclBody(hclBody *hclsyntax.Body) (results []output.Contain
 				for _, arg := range funcExp.Args {
 					re, diags := arg.Value(nil)
 					if diags.HasErrors() {
-						zap.S().Errorf("error fetching containers from aws resource: %v", getErrorMessagesFromDiagnostics(diags))
+						zap.S().Warnf("error fetching containers from aws resource: %v", getErrorMessagesFromDiagnostics(diags))
```

Comment:
@gaurav-gogia let's change this msg as well, instead of `error fetching container`  make it `failed to fetch the container` since we are making it a warning keeping the error message will create confusion. WDYT